common/app/model/content.scala

```diff
@@ -376,7 +377,8 @@ object Article {
       iosType = Some("Article"),
       javascriptConfigOverrides = javascriptConfig,
       opengraphPropertiesOverrides = opengraphProperties,
-      twitterPropertiesOverrides = twitterProperties
+      twitterPropertiesOverrides = twitterProperties,
+      isMinute = content.tags.isUSMinuteSeries
```

Comment:
Ultimately we want to mark a piece of content with both the general Minute identifier and one for the specific topic it covers. Right now all we have is the US campaign minute series tag so we should probably just treat everything as `USMinute` until we have something more general.